stable/app/values.yaml

```
@@ -142,4 +146,23 @@ telegraf:
   command:
     - telegraf
 
-podLabels: {}
\ No newline at end of file
+otelcollector:
+  enabled: true
```

let's don't make it enabled by default
```suggestion
  enabled: false
```